go/vt/vtgate/engine/primitive.go

```diff
@@ -116,6 +116,9 @@ type (
 		MessageStream(rss []*srvtopo.ResolvedShard, tableName string, callback func(*sqltypes.Result) error) error
 
 		VStream(rss []*srvtopo.ResolvedShard, filter *binlogdatapb.Filter, gtid string, callback func(evs []*binlogdatapb.VEvent) error) error
+
+		ShowExec(command sqlparser.ShowCommandType, filter *sqlparser.ShowFilter) (*sqltypes.Result, error)
```

Comment:
nit: it'd be nice with comments here. not entirely self evident what these are for

Reply:
➕ 1️⃣ , 🙏